src/rp2_common/pico_cyw43_driver/cybt_shared_bus/cybt_shared_bus.c

```diff
@@ -62,14 +62,14 @@ static cybt_result_t cybt_fw_download_prepare(uint8_t **p_write_buf, uint8_t **p
     *p_write_buf = NULL;
     *p_hex_buf = NULL;
 
-    *p_write_buf = malloc(BTFW_DOWNLOAD_BLK_SIZE + BTFW_SD_ALIGN);
+    *p_write_buf = cyw43_alloc(BTFW_DOWNLOAD_BLK_SIZE + BTFW_SD_ALIGN);
```

Comment:
If `cyw43_alloc` has the semantics of `malloc` then I'd suggest calling it `cyw43_malloc`.

Reply:
Will do.